Brewfile

```diff
@@ -1,3 +1,14 @@
+def install_ruby_manager
+  # check for existing ruby-install, otherwise install rbenv
+  if system 'command -v ruby-install >/dev/null 2>&1'
+    puts '==> I found ruby-install, so not installing rbenv.'
+  else
+    puts "==> I didn't find a ruby manager, so installing rbenv."
+    brew 'rbenv'
+  end
+end
+
 brew 'postgresql'
 brew 'heroku'
```

Comment:
Can you `brew bundle` here instead, so that we can just add/remove brew dependencies from the `Brewfile`?